core/tests/unit/kmx/meson.build

```diff
@@ -139,7 +139,10 @@ foreach kbd : tests
   cfg.set('CONTENT', content)
 
   kbd_kmn = configure_file(
-    copy: true,
+    # not really a file that needs to be configured, but the `copy` parameter
+    # is deprecated and `fs.copyfile` doesn't work with files that are not in
+    # the source directory
+    configuration: cfg,
```

Comment:
Strange thing is that `fs.copyfile` works in `core/tests/unit/ldml/keyboards/meson.build` where we also use files from outside of the current meson project. I tried but couldn't get it to work here. This is a hack I don't like, but at least it works.